gulpfile.js

```diff
@@ -1,7 +1,10 @@
 const gulp = require('gulp');
```

Comment:
Comments in this would be helpful to make clearer the purpose of each task/constant.